### DISCUSSION TITLE
DiscordMessageFromEvent

### DISCUSSION THREAD

**Discussion by SpokenWig620933** [EssentialsX/Essentials#4580] [Ideas and Feature Suggestions] [answered]
An event that gets executed whenever a message is sent from Discord to Minecraft.
It should be cancellable and have modifiable information on the message and member.

**Comment by SpokenWig620933** [comment]
Any updates on this getting implemented?

**Comment by JRoy** [comment, marked as answer]
Will be added in #4656

**Reply by mdcfe** [reply]
This has now been merged and is available in EssentialsX Discord 2.19.2 and upwards.
